app/src/check-permissions.ts

```diff
@@ -0,0 +1,30 @@
+import api, { route } from '@forge/api';
+
+const isAdmin = async (accountId: any): Promise<boolean> => {
+	const permissionRequestBody = `{
+		"globalPermissions": [
+			"ADMINISTER"
+		],
+		"accountId": "${accountId}"
+	}`;
+
+	const permissions = await api.asApp().requestJira(route`/rest/api/3/permissions/check`, {
```

Comment:
wouldn't the `asApp()` mean we're calling the API as the app itself which has admin access (which was granted to be installed), hence this check will always return true?  I thought this was specifically a check for the user to be an admin?

Reply:
> I feel like we should ask the forge team about admin permissions. I feel like this is something forge should be able to handle out of the box and if not, that's a glaringly missing feature.
> 
> Also, I'd recommend we also implement [displayConditions](https://developer.atlassian.com/platform/forge/manifest-reference/display-conditions/) so that it doesn't show anything to the user if they're not an admin in the first place so they never get a blank page when it fails :)

Currently, if the user is not an admin, they will be unable to access the app anyway. The changes in this PR are to prevent users from calling these functions directly. Assuming the check would happen with `asApp` was the mistake we made initially, that has now been picked up by our security check :)

Reply:
Discussed over zoom the results of the security review and how we knew that Forge hasn't included that security check in `asApp`. All discussed that this route should be `asUser`. Have updated this accordingly.